implementations/rust/ockam/ockam_command/src/transport/create.rs

```diff
@@ -55,7 +55,7 @@ impl CreateCommand {
         // still bad and should be fixed
         let node = command.api_node.unwrap_or_else(|| cfg.get_api_node());
         match command.create_subcommand {
-            CreateTypeCommand::TcpConnector { address } => {
+            CreateTypeCommand::TcpConnector { addr: address } => {
                 cfg.add_transport(&node, false, true, address)
             }
```

Comment:
I think re-mapping the argument name here is not really necessary

Reply:
True, I didn't catch this change after renaming the struct field.